thoth/storages/graph/postgres.py

```diff
@@ -2992,6 +2994,25 @@ def get_software_environments_all(
             result = session.query(software_environment).offset(start_offset).limit(count).all()
             return [model.dict() for model in result]
 
+    def _sort_by(
+        self,
+        query: Query,
+        sort_by: QuerySortTypeEnum
+    ) -> Query:
+        """Add sort to query, which is not executed."""
+        if sort_by.value == QuerySortTypeEnum.PACKAGE_NAME.value:
+            query = query.order_by(PythonPackageVersion.package_name)
+
+        elif sort_by.value == QuerySortTypeEnum.PACKAGE_VERSION.value:
+            raise SortTypeQueryError("To be implemented.")  # TODO: To be implemented
```

Comment:
This will be done after we receive results of the query.

Reply:
So it become a bit tricky to generalize it :thinking: